.github/workflows/ci.yaml

```
@@ -0,0 +1,228 @@
+    needs:
+      - build-pr-image
+    if: |
+      github.event_name == 'pull_request'
+    uses: ./.github/workflows/trivy-scan.yaml
+    with:
+      image_tag: ${{ needs.build-pr-image.outputs.image_tag }}
+    secrets: inherit
+    permissions:
+      security-events: write
+      contents: read
+      secrets: read
```

actionlint (1.7.8)

unknown permission scope "secrets". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)